CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## [Unreleased]
 ### Changed
 - Remove support for Ruby < 2.3
+- Add `Lock::ResilientModern` that uses GET with NX/EX instead of Lua script to set master lock
```

Comment:
Can we link to this PR in the changelog?

Reply:
Sure thing